web/templates/pages/list_mail.html

```diff
@@ -75,146 +140,200 @@
-                  <div class="actions-panel__col actions-panel__edit shortcut-l" key-action="href"><a href="?domain=<?=$key?>&dns=1&token=<?=$_SESSION['token']?>" title="<?=_('DNS records mail');?>"><i class="fas fa-book-atlas status-icon blue status-icon dim"></i></a></div>
-                  <?php if ($data[$key]['SUSPENDED'] == 'no') {?>
-                    <div class="actions-panel__col actions-panel__edit" key-action="href"><a href="http://<?=$webmail;?>.<?=$key?>/" target="_blank" title="<?=_('open webmail');?>"><i class="fas fa-paper-plane status-icon lightblue status-icon dim"></i></a></div>
-                  <?php } ?>
-                <?php } else { ?>
-                  <?php if ($data[$key]['SUSPENDED'] == 'no') {?>
-                    <div class="actions-panel__col actions-panel__logs shortcut-n" key-action="href"><a href="/add/mail/?domain=<?=$key?>&token=<?=$_SESSION['token']?>" title="<?=_('Add Mail Account');?>"><i class="fas fa-circle-plus status-icon green status-icon dim"></i></a></div>
-                    <?php if($_SESSION['WEBMAIL_SYSTEM']){?>
-                      <?php if (!empty($data[$key]['WEBMAIL'])) {?>
-                        <div class="actions-panel__col actions-panel__edit" key-action="href"><a href="http://<?=$webmail;?>.<?=$key?>/" target="_blank" title="<?=_('open webmail');?>"><i class="fas fa-paper-plane status-icon lightblue status-icon dim"></i></a></div>
-                      <?php } ?>
-                    <?php } ?>
-                    <div class="actions-panel__col actions-panel__logs shortcut-enter" key-action="href"><a href="/edit/mail/?domain=<?=$key?>&token=<?=$_SESSION['token']?>" title="<?=_('Editing Mail Domain');?>"><i class="fas fa-pencil status-icon orange status-icon dim"></i></a></div>
-                  <?php } ?>
-                  <div class="actions-panel__col actions-panel__edit shortcut-l" key-action="href"><a href="?domain=<?=$key?>&dns=1&token=<?=$_SESSION['token']?>" title="<?=_('DNS records');?>"><i class="fas fa-book-atlas status-icon blue status-icon dim"></i></a></div>
-                  <div class="actions-panel__col actions-panel__suspend shortcut-s" key-action="js">
-                    <a id="<?=$spnd_action ?>_link_<?=$i?>" class="data-controls do_<?=$spnd_action?>" title="<?=_($spnd_action)?>">
-                      <i class="fas <?=$spnd_icon?> status-icon highlight status-icon dim do_<?=$spnd_action?>"></i>
-                      <input type="hidden" name="<?=$spnd_action?>_url" value="/<?=$spnd_action?>/mail/?domain=<?=$key?>&token=<?=$_SESSION['token']?>">
-                      <div id="<?=$spnd_action?>_dialog_<?=$i?>" class="dialog js-confirm-dialog-suspend" title="<?=_('Confirmation');?>">
-                        <p><?=sprintf($spnd_confirmation,$key)?></p>
-                      </div>
-                    </a>
+    foreach ($data as $key =>
+  $value) { ++$i; if ($data[$key]['SUSPENDED'] == 'yes') { $status = 'suspended'; $spnd_action =
```

Comment:
Before of this is certainly more readable!

Reply:
I think Prettier doesn't like the fact that the templates are html files, but are actually php. It probably loads the html parser instead.

Reply:
Yeah we might need to manually ignore these blocks as we're losing a lot of formatting here.

Reply:
Is there a reason why these files are `.html` instead of `.php`? @jaapmarcus?

Reply:
I don't know I think we "took" it over from Vesta...

Reply:
Should we maybe change it to `.php`? It would be better for syntax highlighting in our IDEs anyways. And it would make Prettier work correctly too.

Reply:
If we want to change it we should do it now. With @AlecRust  class changes auto replace script for adding CSF and so on will break now any way...

Reply:
I can add those to my PR, as well as removing the mobile detect script from below.

Reply:
Go ahead